### DISCUSSION TITLE
How can I prevent placeholder from unmounting after adding a value to Select

### DISCUSSION THREAD

**Discussion by MisterQueen** [JedWatson/react-select#5726] [Q&A]
I would like to implement a placeholder that is similar to material UI.

<img width="236" alt="Знімок екрана 2023-08-15 о 11 24 39" src="https://github.com/JedWatson/react-select/assets/43295416/77996657-892a-4b87-a307-ac3f072505e7">

**Comment by Rall3n** [comment]
@MisterQueen Please do your due diligence and [search](https://github.com/JedWatson/react-select/issues/2152#issuecomment-470889515) [through](https://github.com/JedWatson/react-select/issues/2152#issuecomment-532154264) [existing](https://github.com/JedWatson/react-select/issues/4221) issues before posting an issue/discussion yourself.
